content/docs/rfcs/17/README.md

```diff
@@ -2,127 +2,138 @@
 slug: 17
 title: 17/WAKU-RLN-RELAY
 name: Waku v2 RLN Relay
-status: raw
+status: draft
 tags: waku-core
 editor: Sanaz Taheri <sanaz@status.im>
 ---
 
-The current specification embodies the details of the spam-protected version of `relay` protocol empowered by Rate Limiting Nullifiers (RLN). 
-<!-- More details on RLN can be found in [this spec]() (TODO: to link the spec). -->
+The `17/WAKU-RLN-RELAY` protocol is an extension of `11/WAKU-RELAY` which additionally provides spam protection using [Rate Limiting Nullifiers (RLN)](/spec/32). 
 
-The security objective is to control the number of PubSub messages that each peer can publish per epoch where epoch is a system design parameter, regardless of the published topic.
+The security objective is to contain spam activity in a GossipSub network by enforcing a global messaging rate to all the peers.
+Peers that violate the messaging rate are considered spammers and their message is considered spam.
+Spammers are also financially punished and removed from the system. 
 
 
 **Protocol identifier***: `/vac/waku/waku-rln-relay/2.0.0-alpha1`
```

Comment:
Is this still the protocol identifier? Have we ever bumped this?

Reply:
Good point, In fact, we do not use the waku-rln-relay protocol identifier at all and I am going to delete it from the specs. Rln-relay is a configuration over the `relay` protocol that applications can switch on-off for specific pubsub topics. More specifically, rln-relay is a `topicValidator` that can be installed on a `pubsub topic` if need be.